wsf_scraping/spiders/nice_spider.py

```diff
@@ -0,0 +1,227 @@
+import os
+import json
+import scrapy
+import logging
+from lxml import html
+from scrapy.http import Request
+from wsf_scraping.items import NICEArticle
+from tools.dbTools import check_db, is_scraped
+
+
+class NiceSpider(scrapy.Spider):
+    name = 'nice'
+
+    def start_requests(self):
+        """Set up the initial request to the website to scrape."""
+
+        urls = []
+        articles_count = self.settings['NICE_ARTICLES_COUNT']
+
+        # Initial URL (splited for PEP8 compliance). -1 length displays
```

Comment:
I think you can also carry on in a new line if you break \

https://stackoverflow.com/questions/1874592/how-to-write-very-long-string-that-conforms-with-pep8-and-prevent-e501

Reply:
Doesn't works well with urls. I should have put it in a ''.join(), though. I'll change it 👍

Reply:
I know, but I found it more readable this way. I can change it back, if you don't.

Reply:
up to you